### PR TITLE
Need help with routing

### PR DESCRIPTION
**Pull request by pawlyk**
Hi!
I need some help with routing and js in templates.
When I change 
`https://github.com/jettify/aiohttp_admin/blob/master/aiohttp_admin%2F__init__.py#L31` 
to `r.add_route('GET', '/admin/config/js/config.js', admin.config_handler, name='admin.config')` and 
`https://github.com/jettify/aiohttp_admin/blob/master/aiohttp_admin%2Ftemplates%2Fadmin.html#L14` 
to `<script src="{{ url('admin.config') }}" type="text/javascript"></script>` 
everything works but its awkward way. 
I need a little help with it.

### CODE REVIEW

**Comment by jettify** [comment]
1) Lets do not do any routing and integration right now, just simple utils to build template
2) you may want to use other config.js (for instance https://github.com/jettify/aiohttp_admin/blob/master/demos%2Fpolls%2Fstatic%2Fjs%2Fconfig.js) for template, because yours have bunch of custom stuff

**Comment by jettify** [comment]
For the first release I do not want to expose generated `config.js` over api, but use it as initial start for mapping admin interface. Lets move config generation into separate file for now

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,2 @@
+recursive-include aiohttp_admin/static *
+recursive-include aiohttp_admin/templates *
```
Comment (jettify) [inline MANIFEST.in]: good point. Could you add this changes in separate PR?

**Comment by pawlyk** [comment]
Ok. Thanx.

**Pull request closed without merging**